agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1489,6 +1499,15 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<SSlaveSocke
             }
         }
 
+        if (m_eFSMState >= EState::CONNECT_TO_MASTER) {
+            LOG(INFO) << "Sending topology notification to the controller on reconnected son_slave";
+            if (!cmdu_tx.create(0, ieee1905_1::eMessageType::TOPOLOGY_NOTIFICATION_MESSAGE)) {
+                LOG(ERROR) << "cmdu creation of type TOPOLOGY_NOTIFICATION_MESSAGE, has failed";
+                return false;
+            }
+            return send_cmdu_to_bus(cmdu_tx, controller_bridge_mac, bridge_info.mac);
+        }
```

Comment:
Same remarks as above. Maybe move the entire thing into a function (including the state condition).

Reply:
Changed to relayed multicast, but I prefer to not turning it to a function since it is not a lot of code and I would like to see the print with the right location.